recipe/meta.yaml

```diff
@@ -30,39 +29,39 @@ requirements:
     - pip
     - nodejs 18.15.0
     - hatch-jupyter-builder
-    - hatchling
+    - hatchling >=1.11
     - python
-    - setuptools
-    - wheel
   run:
     - python
     - anyio >=3.1.0,<4
```

Comment:
The upper bound seems weird to me.

Reply:
Addressed. for ref: https://github.com/jupyter-server/jupyter_server/pull/1192